src/env.cc

```diff
@@ -23,6 +24,8 @@ using v8::Message;
 using v8::StackFrame;
 using v8::StackTrace;
 
+Environment *Environment::currentEnvironment;
```

Comment:
style nit: align the `*` to the left (i.e. `Environment* Environment::currentEnvironment`)